iset.mm

```diff
@@ -153080,6 +153114,230 @@ the additive structure must be abelian (see ~ ringcom ), care must be
+    cz cnfldmul mp2an ) MABCABDEFGHIJMCEDAAKNLO $.
+
+  $( The zero element of the ring of integers.  (Contributed by Thierry Arnoux,
+     1-Nov-2017.)  (Revised by AV, 9-Jun-2019.) $)
+  zring0 $p |- 0 = ( 0g ` ZZring ) $=
+    ( ccnfld cmnd wcel cc0 cz wss czring c0g cfv wceq ccrg crg crngring ringmnd
+    cc cncrng mp2b 0z zsscn df-zring cnfldbas cnfld0 ress0g mp3an ) ABCZDECEOFD
+    GHIJAKCALCUEPAMANQRSEOAGDTUAUBUCUD $.
+
+  $( The unity element of the ring of integers.  (Contributed by Thierry
+     Arnoux, 1-Nov-2017.)  (Revised by AV, 9-Jun-2019.) $)
+  zring1 $p |- 1 = ( 1r ` ZZring ) $=
+    ( cz ccnfld csubrg cfv wcel c1 czring cur wceq zsubrg df-zring cnfld1 ax-mp
+    subrg1 ) ABCDEFGHDIJABGFKLNM $.
+
+  $( The ring of integers is a nonzero ring.  (Contributed by AV,
+     18-Apr-2020.) $)
+  zringnzr $p |- ZZring e. NzRing $=
+    ( czring cnzr wcel crg cc0 wne zringring 1ne0 zring1 zring0 isnzr mpbir2an
+    c1 ) ABCADCMEFGHAMEIJKL $.
+
+  ${
+    $d x y z $.
+    $( Ring divisibility in the ring of integers corresponds to ordinary
+       divisibility in ` ZZ ` .  (Contributed by Stefan O'Rear, 3-Jan-2015.)
+       (Revised by AV, 9-Jun-2019.) $)
+    dvdsrzring $p |- || = ( ||r ` ZZring ) $=
+      ( vx vy vz cv cz wcel wa cmul wceq wrex copab cdvds czring cdsr cfv simpl
+      co anim1i wtru a1i zmulcl ancoms eleq1 syl5ibcom rexlimdva impbii opabbii
+      imp simpr jca31 df-dvds cbs zringbas crg csrg zringring ringsrg zringmulr
+      eqidd mp1i cmulr dvdsrvald mptru 3eqtr4i ) ADZEFZBDZEFZGZCDZVEHQZVGIZCEJZ
+      GZABKVFVMGZABKZLMNOZVNVOABVNVOVIVFVMVFVHPRVOVFVHVMVFVMPVFVMVHVFVLVHCEVFVJ
+      EFZGVKEFZVLVHVRVFVSVJVEUAUBVKVGEUCUDUEUHVFVMUIUJUFUGABCUKVQVPISABCEVQMHEM
+      ULOISUMTSVQUSMUNFMUOFSUPMUQUTHMVAOISURTVBVCVD $.
+  $}
+
+  $( The additive inverse of an element of the ring of integers.  (Contributed
+     by AV, 24-May-2019.)  (Revised by AV, 10-Jun-2019.) $)
+  zringinvg $p |- ( A e. ZZ -> -u A = ( ( invg ` ZZring ) ` A ) ) $=
+    ( cz wcel czring cminusg cfv cneg wceq caddc co cc0 negidd cgrp wb zringgrp
+    zcn id znegcl zringbas zringplusg zring0 grpinvid1 mp3an2i mpbird eqcomd
+    eqid ) ABCZADEFZFZAGZUGUIUJHZAUJIJKHZUGAAPLDMCUGUGUJBCUKULNOUGQARBIDUHAUJKS
+    TUAUHUFUBUCUDUE $.
+
+  ${
+    zringsubgval.m $e |- .- = ( -g ` ZZring ) $.
+    $( Subtraction in the ring of integers.  (Contributed by AV,
+       3-Aug-2019.) $)
+    zringsubgval $p |- ( ( X e. ZZ /\ Y e. ZZ ) -> ( X - Y ) = ( X .- Y ) ) $=
+      ( cz ccnfld csubg cfv wcel cmin wceq csubrg zsubrg subrgsubg ax-mp czring
+      co cnfldsub df-zring subgsub mp3an1 ) EFGHIZBEICEIBCJQBCAQKEFLHIUBMEFNOEF
+      PJABCRSDTUA $.
+  $}
+
+  $( The multiplication group of the ring of integers is the restriction of the
+     multiplication group of the complex numbers to the integers.  (Contributed
+     by AV, 15-Jun-2019.) $)
+  zringmpg $p |- ( ( mulGrp ` CCfld ) |`s ZZ ) = ( mulGrp ` ZZring ) $=
```

Comment:
A carryover typo:
multiplication group --> multiplicative group

Reply:
Changed in both set.mm and iset.mm.